app/src/test/java/com/example/benchproject/domain/popular/movies/usecase/GetPopularMoviesUseCaseTest.kt

```diff
@@ -0,0 +1,56 @@
+package com.example.benchproject.domain.popular.movies.usecase
+
+import com.example.benchproject.domain.popular.movies.entity.MovieModel
+import com.example.benchproject.domain.popular.movies.repo.PopularMoviesRepository
+import io.mockk.coEvery
+import io.mockk.mockk
+import kotlinx.coroutines.test.runTest
+import org.amshove.kluent.shouldBe
+import org.amshove.kluent.shouldBeEqualTo
+import org.junit.jupiter.api.Test
+
+internal class GetPopularMoviesUseCaseTest {
+
+    private val movieListModel = Result.success(
+        listOf(
+            MovieModel(10001, "Test1", 5.4, "2022-06-23", "https:/something1.com"),
+            MovieModel(10002, "Test2", 6.4, "2022-06-22", "https:/something2.com"),
+            MovieModel(10003, "Test3", 3.4, "2022-06-21", "https:/something3.com")
+        )
+    )
+    private val badResponseFromRepository =
+        Result.failure<List<MovieModel>>(
+            Throwable(
```

Comment:
New line redundant here, message could be in one line

Reply:
Ok I deleted this new line and now it looks like 
```
private val badResponseFromRepository =
        Result.failure<List<MovieModel>>(
            Throwable("Unable to resolve host \"api.themoviedb.org\": No address associated with hostname")
        )
```